fxtest/fxtest.go

```diff
@@ -107,10 +106,13 @@ func NewLifecycle(t TB) *Lifecycle {
 // hook that doesn't succeed.
 func (l *Lifecycle) Start(ctx context.Context) error { return l.lc.Start(ctx) }
 
-// RequireStart calls Start with context.Background(), failing the test if an
+// RequireStart calls Start with context.Background() defering a cancel, failing the test if an
```

Comment:
nit: spelling of "deferring." Also, not sure that this actually needs to be documented - *all* cancel functions should be called, so the previous behavior was just a bug.

Reply:
ok removed my extra bit - it was a bit awkward i admit.